turbo.json

```diff
@@ -1,6 +1,7 @@
 {
   "$schema": "https://turborepo.org/schema.json",
   "globalEnv": ["NEXT_CI_RUNNER"],
+  "daemon": false,
```

Comment:
> Good to know:When running in a CI environment the daemon is always disabled regardless of this setting.

https://turbo.build/repo/docs/reference/configuration#daemon

Reply:
The hypothesis in slack was that this detection is failing causing daemon to be used